hw/drivers/sensors/adxl345/src/adxl345_shell.c

```diff
@@ -146,67 +147,67 @@ adxl345_shell_cmd_dump(int argc, char **argv)
+    console_printf("0x%02X (THRESH_FF): 0x%02X %s\n", ADXL345_THRESH_FF, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_TIME_FF, &val);
+    console_printf("0x%02X (TIME_FF): 0x%02X %s\n", ADXL345_TIME_FF, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_TAP_AXES, &val);
+    console_printf("0x%02X (TAP_AXES): 0x%02X %s\n", ADXL345_TAP_AXES, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_ACT_TAP_STATUS, &val);
+    console_printf("0x%02X (ACT_TAP_STATUS): 0x%02X %s\n", ADXL345_ACT_TAP_STATUS, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_BW_RATE, &val);
+    console_printf("0x%02X (BW_RATE): 0x%02X %s\n", ADXL345_BW_RATE, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_POWER_CTL, &val);
+    console_printf("0x%02X (POWER_CTL): 0x%02X %s\n", ADXL345_POWER_CTL, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_INT_ENABLE, &val);
+    console_printf("0x%02X (INT_ENABLE): 0x%02X %s\n", ADXL345_INT_ENABLE, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_INT_MAP, &val);
+    console_printf("0x%02X (INT_MAP): 0x%02X %s\n", ADXL345_INT_MAP, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_INT_SOURCE, &val);
+    console_printf("0x%02X (INT_SOURCE): 0x%02X %s\n", ADXL345_INT_SOURCE, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATA_FORMAT, &val);
+    console_printf("0x%02X (DATA_FORMAT): 0x%02X %s\n", ADXL345_DATA_FORMAT, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAX0, &val);
+    console_printf("0x%02X (DATAX0): 0x%02X %s\n", ADXL345_DATAX0, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAX1, &val);
+    console_printf("0x%02X (DATAX1): 0x%02X %s\n", ADXL345_DATAX1, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAY0, &val);
+    console_printf("0x%02X (DATAY0): 0x%02X %s\n", ADXL345_DATAY0, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAY1, &val);
+    console_printf("0x%02X (DATAY1): 0x%02X %s\n", ADXL345_DATAY1, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAZ0, &val);
+    console_printf("0x%02X (DATAZ0): 0x%02X %s\n", ADXL345_DATAZ0, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_DATAZ1, &val);
+    console_printf("0x%02X (DATAZ1): 0x%02X %s\n", ADXL345_DATAZ1, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_FIFO_CTL, &val);
+    console_printf("0x%02X (FIFO_CTL): 0x%02X %s\n", ADXL345_FIFO_CTL, val, rc ? " read failed": "");
+    rc = adxl345_read8(&g_sensor_itf, ADXL345_FIFO_STATUS, &val);
+    console_printf("0x%02X (FIFO_STATUS): 0x%02X %s\n", ADXL345_FIFO_STATUS, val, rc ? " read failed": "");
```

Comment:
It might be more helpful to printing the value of `rc` rather than a "success" / "failure" indication.